src/us03.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import copy
+import numpy as np
+import pandas as pd
+import datetime
+import os
+sys.path.append(os.path.abspath('../src/seeds'))
+import Project02
+
+#user03 checks to see if birth dates are before death dates.
+def user03(gedcom_file):
```

Comment:
nit: function name should match file name

Reply:
changed the name